src/main/config/config.c

```diff
@@ -475,10 +475,7 @@ static void validateAndFixConfig(void)
     featureDisableImmediate(FEATURE_RSSI_ADC);
 #endif
 
-// Enable features in Cloud Build
-#ifdef CLOUD_BUILD
-
-if (systemConfig()->configurationState == CONFIGURATION_STATE_DEFAULTS_BARE) {
+if (systemConfig()->configurationState != CONFIGURATION_STATE_CONFIGURED) {
```

Comment:
Testing for 'unconfigured' may be slightly more robust when other states are added
```suggestion
if (systemConfig()->configurationState == CONFIGURATION_STATE_UNCONFIGURED) {
```

Reply:
Thanks!